recipes/sirocco/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf --install
+./configure --prefix=$PREFIX --libdir=$PREFIX/lib
+make -j${CPU_COUNT}
+make install
```

Comment:
Add `make check` after this?

Reply:
This is my first conda package (and not really mine) and I followed what you did for libbraiding; I agree with all the changes